assets/shared/structure/structure-store.js

```diff
@@ -28,149 +29,207 @@ export function registerStructureStore( {
 	storeName,
 	getEndpoint,
 	saveError,
+	fetchError,
```

Comment:
Not critical, but I think a separation of the actions in a separate property could organize it a little better. And since some of the `opts` are actions, we could add them to the actions object too. Does it make sense to you too?

Reply:
Yeah I was thinking about this. At the end, these functions are not really actions or action generators for the store. They are external side-effects/integrations. Maybe we could bring them closer to wp-data as controls, but that would need a lot of extra indirection and complexity.
Also everything but the `storeName` would be grouped in the actions/controls object.